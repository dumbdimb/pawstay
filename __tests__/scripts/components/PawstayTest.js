jest.unmock('../../../src/scripts/components/Pawstay');

import React from 'react';
import {shallow} from 'enzyme';
import renderer from 'react-test-renderer';

import Pawstay from '../../../src/scripts/components/Pawstay';

describe('Pawstay', () => {
    let tree;

    beforeEach(() => {
        tree = shallow(<Pawstay/>);
    });

    it('renders coorectly', () => {
        expect(renderer.create(<Pawstay/>).toJSON()).toMatchSnapshot();
    });

    describe('changePage', () => {
        it('changes currentPage to the parameter with which it is called', () => {
            tree.find('.root__toolbar').props().changePage('services');

            expect(tree.update().state('currentPage')).toEqual('services');
        });
    });
});