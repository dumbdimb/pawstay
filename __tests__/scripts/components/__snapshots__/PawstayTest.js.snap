// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pawstay renders coorectly 1`] = `
<div
  className="root"
>
  <div
    className="toolbar"
  >
    <div
      className="toolbar__content"
    >
      <div
        className="toolbar__logo"
        onClick={[Function]}
      />
      <div
        className="toolbar__menu-item"
        onClick={[Function]}
      >
        about us
      </div>
      <div
        className="toolbar__menu-item"
        onClick={[Function]}
      >
        our service
      </div>
      <div
        className="toolbar__menu-item"
        onClick={[Function]}
      >
        contact us
      </div>
    </div>
  </div>
  <div
    className="home"
  >
    <div
      className="home__blurb"
    >
      <div
        className="home__blurb__hello"
      >
        hello
      </div>
      <div
        className="home__blurb__who-we-are"
      >
        we are
      </div>
      <div
        className="home__blurb__who-we-are"
      >
        a personal owned and home based
      </div>
      <div
        className="home__blurb__who-we-are"
      >
        dog boarding and daycare
      </div>
    </div>
    <div
      className="home__darby"
    />
  </div>
</div>
`;
